libs/etterlatte-funksjonsbrytere/build.gradle.kts

```diff
@@ -7,5 +7,5 @@ repositories {
 }
 
 dependencies {
-    implementation(libs.unleash.client)
+    api(libs.unleash.client)
```

Comment:
dette er fordi alle brukere av etterlatte-funksjonsbrytere ellers må dra inn `implementation(libs.unleash.client)`

Eventuelt skrive en egen variant av contexten, men det synes jeg er helt unødvendig for noe slikt som Unleash, som man bare forventer at er tilgjengelig fra overalt i koden mer eller mindre.